mod_ci/controllers.py

```diff
@@ -573,6 +574,16 @@ def start_ci():
             if issue is not None:
                 issue.title = issue_data['title']
                 issue.status = issue_data['state']
+                # Send Email to the Mailing List using the Mailer Module and Mailgun's API
+                issue_id = issue_data['number']
```

Comment:
This is in the wrong spot. We want to send an email on every opened issue. If you put it inside this check, it'll only send out emails when the issue is in our DB (which only happens when a user submits the issue through the platform).

You'll have to move up the code ;)